src/tests/ftest/util/dmg_utils_base.py

```diff
@@ -6,8 +6,10 @@
 from socket import gethostname
 
 from ClusterShell.NodeSet import NodeSet
+
 from command_utils import CommandWithSubCommand, YamlCommand
-from command_utils_base import BasicParameter, CommandWithParameters, FormattedParameter
+from command_utils_base import (BasicParameter, CommandWithParameters,
+                                FormattedParameter)
 from environment_utils import TestEnvironment
```

Comment:
> Imports are incorrectly sorted and/or formatted.

I think it wants this
```suggestion
from ClusterShell.NodeSet import NodeSet
from command_utils import CommandWithSubCommand, YamlCommand
from command_utils_base import BasicParameter, CommandWithParameters, FormattedParameter
```

Reply:
Sure.